kong/plugins/key-auth/migrations/init.lua

```diff
@@ -1,4 +1,5 @@
 return {
   "000_base_key_auth",
   "001_14_to_15",
+  "002_add_signature_validation"
```

Comment:
Please respect the naming scheme for new migrations: version to version, and not "feature-centric". All migrations required between two versions are now part of the same migration.